README.md

```diff
@@ -126,6 +156,13 @@ Write code to do the following:
 Write your code here:
 ```ruby
 # code here
+lamp_one = Lamp.create(wishes_remaining: 3)
+genie_one = Genie.create(name: "Genie")
+lamp_one.update(genie_id: genie_one.id)
+lamp_one(wishes_remaining: 1)
+jafar = Genie.create(name: "Jafar")
+jafar.lamps.create(wishes_remaining: 3)
```

Comment:
because of our relationship where a `lamp` has_many `genie`(s), we have to be careful of our pluratity: 
i.e:
```ruby
jafar = Genie.create(name: "Jafar")
jafar.lamp.create(wishes_remaining: 3)
```